pegasus/rake/seed.rake

```diff
@@ -94,6 +94,7 @@ class CsvToSqlTable
       '_dt' => {type: 'datetime'},
       '_f' => {type: 'float'},
       '_i' => {type: 'integer'},
+      '_bi' => {type: 'bigint'},
```

Comment:
Thanks. I hadn't noticed that it does some special casing of types in `hash_from_keys_and_values` (for a subset of supported types). It looks like we are hitting the fall through here (https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/rake/seed.rake#L58) and just assigning without any conversion. However, this turns out to work - I get the correct 7.4B result in my local DB. Especially since only a few types are singled out here, I think its OK the way it is. I have looked further and there doesn't appear to be anything else that would need to be updated.